packages/health/ios/Classes/SwiftHealthPlugin.swift

```diff
@@ -1153,16 +1195,48 @@ public class SwiftHealthPlugin: NSObject, FlutterPlugin {
                 forIdentifier: .bodyFatPercentage)!
             dataTypesDict[BODY_MASS_INDEX] = HKSampleType.quantityType(forIdentifier: .bodyMassIndex)!
             dataTypesDict[BODY_TEMPERATURE] = HKSampleType.quantityType(forIdentifier: .bodyTemperature)!
-            dataTypesDict[DIETARY_CARBS_CONSUMED] = HKSampleType.quantityType(
-                forIdentifier: .dietaryCarbohydrates)!
-            dataTypesDict[DIETARY_ENERGY_CONSUMED] = HKSampleType.quantityType(
-                forIdentifier: .dietaryEnergyConsumed)!
-            dataTypesDict[DIETARY_CAFFEINE] = HKSampleType.quantityType(
-                forIdentifier: .dietaryCaffeine)!
-            dataTypesDict[DIETARY_FATS_CONSUMED] = HKSampleType.quantityType(
-                forIdentifier: .dietaryFatTotal)!
-            dataTypesDict[DIETARY_PROTEIN_CONSUMED] = HKSampleType.quantityType(
-                forIdentifier: .dietaryProtein)!
+            
+            // Nutrition
+            dataTypesDict[DIETARY_CARBS_CONSUMED] = HKSampleType.quantityType(forIdentifier: .dietaryCarbohydrates)!
```

Comment:
All of the micronutrients are available in ios 8+ besides dietaryWater which is ios 9+
I believe the plugin only differentiates for ios 11 and 13, so it should be safe to add these always